pkg/api/types.go

```diff
@@ -144,18 +157,20 @@ type InstanceSpec struct {
 	// CF-specific; move to annotations
 	CFCredentials   string
 	CFDashboardURL  string
-	CFInternalID    string // came from ville. remove? nobody likes that guy
+	CFInternalID    string
 	CFServiceID     string
 	CFPlanID        string
 	CFType          string
 	CFSpaceGUID     string
 	CFLastOperation string
 }
 
+// InstanceStatus represents the current status of an Instance.
 type InstanceStatus struct {
 	Conditions []InstanceCondition
 }
 
+// InstanceCondition represents an aspect of an Instance's status.
 type InstanceCondition struct {
```

Comment:
Should we have a common typedef for these structs? Won't all *Condition things follow the same structure?